src/Integration/Binding/BindingController.cs

```diff
@@ -148,26 +148,7 @@ void IBindingWorkflowExecutor.BindProject(BindCommandArgs bindingArgs)
                     Strings.Bind_FirstTimeBinding :
                     Strings.Bind_UpdatingNewStyleBinding);
 
-            var solutionBindingOp = new SolutionBindingOperation();
```

Comment:
Moved this ugly code out of the `BindingContoller` into a separate factory class.
There are no other product changes.

This is part of a series of changes I'm planning to make it possible to create an `IBindingProcess` independently of the Team Explorer page and the `IHost`.

The idea is that we will be able to create and execute the binding steps in the background, whether Team Explorer page exists or not.